joss_paper/paper.md

```diff
@@ -0,0 +1,221 @@
+---
+title: 'Zarrtraj: A Python package for streaming molecular dynamics trajectories from cloud services'
+tags:
+  - streaming
+  - molecular-dynamics
+  - file-format
+  - mdanalysis
+  - zarr
+authors:
+  - name: Lawson Woods
+    orcid: 0009-0003-0713-4167
+    affiliation: [1, 2]
+  - name: Hugo MacDermott-Opeskin
+    orcid: 0000-0002-7393-7457
+    affiliation: [3]
+  - name: Edis Jakupovic 
+    affiliation: [4, 5]
+  - name: Yuxuan Zhuang
+    orcid: 0000-0003-4390-8556
+    affiliation: [6, 7]
+  - name: Richard J Gowers
+    orcid: 0000-0002-3241-1846
+  - name: Oliver Beckstein
+    orcid: 0000-0003-1340-0831
+    affiliation: [4, 5]
+affiliations:
+ - name: School of Computing and Augmented Intelligence, Arizona State University, Tempe, Arizona, United States of America
+   index: 1
+ - name: School of Molecular Sciences, Arizona State University, Tempe, Arizona, United States of America
+   index: 2
+ - name: Open Molecular Software Foundation, Davis, CA, United States of America
+   index: 3
+ - name: Center for Biological Physics, Arizona State University, Tempe, AZ, United States of America
+   index: 4
+ - name: Department of Physics, Arizona State University, Tempe, Arizona, United States of America
+   index: 5
+ - name: Department of Computer Science, Stanford University, Stanford, CA 94305, USA.
+   index: 6
+ - name: Departments of Molecular and Cellular Physiology and Structural Biology, Stanford University School of Medicine, Stanford, CA 94305, USA.
+   index: 7
+date: 23 October 2024
+bibliography: paper.bib
+---
+
+# Summary
+
+Molecular dynamics (MD) simulations provide a microscope into the behavior of 
+atomic-scale environments otherwise prohibitively difficult to observe. However,
+the resulting trajectory data are too often siloed in a single institutions' 
+HPC environment, rendering it unusable by the broader scientific community.
```

Comment:
This is true and a nice FAIR story, but another interesting angle is that on-prem storage isn't as ubiquitous as it used to be, some places have all their data in the cloud.  Possibly you could insert a sentence as below that would include this and not break the overall flow

"Additionally it is increasingly common for trajectory data to be entirely stored in a cloud storage provider, rather than a traditional on-premise storage site"